### PR TITLE
[nfs] merge nfsserver plugin into nfs one

### PR DESCRIPTION
**Pull request by pmoravec**
nfsserver plugin enabledness relies on legacy init scripts. A more
appropriate way to idenfity a NFS server is via nfs-utils package
(though it could be present also on a NFS client).

As that package enables nfs plugin, it is reasonable to merge the
plugins into one.

Resolves: #2061

Signed-off-by: Pavel Moravec <pmoravec@redhat.com>

---
Please place an 'X' inside each '[]' to confirm you adhere to our [Contributor Guidelines](https://github.com/sosreport/sos/wiki/Contribution-Guidelines)

- [X] Is the commit message split over multiple lines and hard-wrapped at 72 characters?
- [X] Is the subject and message clear and concise?
- [X] Does the subject start with **[plugin_name]** if submitting a plugin patch or a **[section_name]** if part of the core sosreport code?
- [X] Does the commit contain a **Signed-off-by: First Lastname <email@example.com>**?
- [X] If this commit closes an existing issue, is the line `Closes: #ISSUENUMBER` included in an independent line?
- [X] If this commit resolves an existing pull request, is the line `Resolves: #PRNUMBER` included in an independent line?

### CODE REVIEW

**Comment by pmoravec** [comment]
Cc @pierg75 for review and @BryanQuigley for Nfs plugin extension on Ubuntu/Debian - wont these extra files/dirs and commands harm?

**Comment by pierg75** [comment]
Looks good to me. Thanks!

File: sos/report/plugins/nfs.py [hunk 1]
```diff
@@ -14,7 +14,7 @@ class Nfs(Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin):
     short_desc = 'Network file system information'
     plugin_name = 'nfs'
     profiles = ('storage', 'network', 'nfs')
-    packages = ['nfs-utils']
+    packages = ('nfs-utils')
```
Comment (TurboTurtle) [inline sos/report/plugins/nfs.py:17]: Needs to be a tuple

**Comment by pmoravec** [comment]
Ah, I see, fixed.

**Pull request closed without merging**